src/main/java/io/outfoxx/swiftpoet/CodeWriter.kt

```diff
@@ -198,13 +198,16 @@ internal class CodeWriter constructor(
 
   fun emitCode(format: String, vararg args: Any?) = emitCode(CodeBlock.of(format, *args))
 
-  fun emitCode(codeBlock: CodeBlock) = apply {
+  fun emitCode(
+    codeBlock: CodeBlock,
+    isConstantContext: Boolean = false,
```

Comment:
Can we have a quick comment to `isConstantContext` to say what it does?

Reply:
Its when outputting a string as a constant. This is helpful for attributes etc. It is copied from the KotlinPoet API but the same applies here for Swift.

When outputting Attribute specs we treat those as constants e.g. `@MyDefaultValue(value: "string")`